frequently-asked-questions/lending.md

```diff
@@ -59,3 +61,49 @@ The THORChain protocol and all RUNE holders are the counterparty to each loan. T
 ### How do I integrate with Lending?
 
 See the developer quick start guide [here](https://dev.thorchain.org/lending/quick-start-guide.html)
+
+## In Depth Questions
+
+[Source](https://twitter.com/CBarraford/status/1775618463341408587)
+
+### No liquidation? Only a matter of time before loans are insolvent and the protocol collapses
+
+The design of THORChain Lending is very different than defi loans you’ve seen in the past, and it operates on a different set of assumptions. The need for liquidation is typically to ensure the protocol remains solvent, but that is not needed here. In fact, when collateral > debt, its a good thing for the protocol as it REMOVES liability. This is because the protocol already “made money” when the loan was opened, and now the loan won’t be closed (so the protocol had only upside and no downside). Even if the loan was closed in this state, the protocol would still net profit, 100% guaranteed (due to buy/burn the debt value and sell/mint the collateral value).
+
+### There's no risk? Sounds like a scam
+
+There certainly is risk, and no one is claiming otherwise. You always take on protocol risk for example, and it has an opportunity cost (ie you could be earning 2-6% APR on savers, but instead using your BTC for loan collateral). There is also a chance of high fees when opening or closing your loan depending on market conditions. So you may not be able to close your loan when you want to (at least not with low fees).
+
+### Seen it before…TC will rug just like blockfi, celsius and all the others
+
+CeFi companies like BlockFi Celsius etc took people’s collateral and made increasingly riskier bets to compete in a field of “high yielders” offerings. THORChain isn’t doing this, instead, it does only one thing with you collateral which is transparent and publicly known what that is exactly. While there is protocol risk (ie TSS exploit where funds are lost), it has been several years since an exploit has been carried out on the protocol, which instils confidence in the protocol and the devs/security engineers who work on it.
+
+### What if there's a bank run?
```

Comment:
Don't feel comfortable approving this language in official documentation

Reply:
Drop this FAQ and I'll approve as-is. This kind of literature belong somewhere else

Reply:
Seen it before… removed